pcapng/utils.py

```diff
@@ -1,12 +1,20 @@
 import socket
 import struct
+from collections import namedtuple as _namedtuple
 
-import six
-from six import byte2int
+# These are here for export, hence ignoring F401 warning
```

Comment:
Avoid the re-exporting, particularly as part of `utils`.

If you ever need to create re-exports for compatibility between version, consider creating a `_compat.py` file, that way it's clear that it is not part of the module's interface.